src/components/FeeExplainer/FeeExplainer.tsx

```diff
@@ -300,11 +304,13 @@ export const FeeOutput: React.FC<FeeOutputProps> = ({ tradeSizeUSD, foxHolding,
     [feeUsdBeforeDiscount, feeBpsBeforeDiscount],
   )
 
-  const isFree = useMemo(() => bnOrZero(feeUsd).lte(0), [feeUsd])
+  const isFree = useMemo(() => bnOrZero(feeUsd).lte(0) || isFoxBuyAsset, [feeUsd, isFoxBuyAsset])
 
   const discountTypeTranslation = useMemo(() => {
+    if (isFoxBuyAsset) return translate('foxDiscounts.foxBuy')
```

Comment:
preferably-blocking: we probably want to check that there *were* some bps here, either inline, or preferably contain the logic within `appliedDiscountType`.

Else this will currently show up as "FOX Buy" discount, whereas there were no bps to start with:

<img width="537" alt="image" src="https://github.com/user-attachments/assets/5f61ac31-fbfb-4bd1-b8d7-28a692772097" />


Also while at that, noticed that the applied discount type is *not* translated on the simulation tab, but is correctly translated in the "Fee Summary" one, can we fix this while at that? (bug present on prod already)

<img width="552" alt="image" src="https://github.com/user-attachments/assets/4dce0b2a-aefa-4cba-8b88-a8e6dd66c92f" />

Reply:
![image](https://github.com/user-attachments/assets/8d51e676-2a42-406f-8b46-849b52d74488)
With no bps:
![image](https://github.com/user-attachments/assets/52636233-2402-4803-b2af-b6171ad2cab0)

Fixed in [3184e2f](https://github.com/shapeshift/web/pull/8965/commits/3184e2f9f17b44905f04e18cc9d24ce78459d1ed)